libs/partners/together/pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [tool.poetry]
 name = "langchain-together"
-version = "0.1.0"
-description = "An integration package connecting Together and LangChain"
+version = "0.3.0"
```

Comment:
Is this a breaking change? Could we do `0.1.1` instead? Any reason we're skipping 0.2?

Reply:
You're right, not a breaking change! Will change to 0.1.1